### ISSUE TITLE
Requirements.txt

### ISSUE THREAD

**Issue by dacieri** [apollo-lhc/CM_FPGA_FW#48]
Hi, 
I think it would be nice to have a requirements.txt in the top repository folder, listing all the python packages required to compile the code, with the supported version. I noticed for instance that the code cannot be compiled with the latest version of `pyyaml`(>=5.4), since the `load` function now requires also the `Loader` to be specified automatically.

**Comment by pwittich** [comment]
This sounds like a good solution to this issue. FWIW, I found that you could fix the immediate issue by replacing `load` with `safe_load`, [as per this SO post.](https://stackoverflow.com/questions/69564817/typeerror-load-missing-1-required-positional-argument-loader-in-google-col)

**Comment by dgastler** [comment]
Yeah, there is a branch where this is fixed (found by @pwittich ), there is a package list in the README.md, but it lacks versions.
